app/src/androidTest/kotlin/cz/covid19cz/erouska/screenObject/FinishActivation.kt

```diff
@@ -0,0 +1,25 @@
+package cz.covid19cz.erouska.screenObject
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import cz.covid19cz.erouska.R
+import org.awaitility.Awaitility.await
+import java.util.concurrent.TimeUnit
+
+class FinishActivation {
+    companion object {
+        const val TITLE = "eRouška potřebuje běžet i když s ní právě nepracujete"
```

Comment:
Asi zbytečný na tohle dělat konstanty, když to je stejně hardcoded. Čitelnosti to nepomůže. Dal bych rovnou do kódu a později by se to mělo předělat aby to umělo víc jazyků.